package.json

```diff
@@ -27,50 +27,51 @@
     ]
   },
   "devDependencies": {
-    "eslint": "^7.3.1",
-    "jest": "^26.1.0",
+    "eslint": "^7.17.0",
+    "jest": "^26.6.3",
     "sass": "^1.26.10",
     "typescript": "^4.0.2",
     "webpack": "^4.35.0",
     "webpack-dev-server": "^3.7.2"
   },
   "dependencies": {
-    "@babel/core": "^7.10.3",
-    "@babel/plugin-proposal-class-properties": "^7.10.1",
-    "@wordpress/babel-preset-default": "^4.16.0",
-    "babel-loader": "^8.0.6",
+    "@babel/core": "^7.12.10",
+    "@babel/plugin-proposal-class-properties": "^7.12.1",
+    "@wordpress/babel-preset-default": "^4.20.0",
+    "babel-loader": "^8.2.2",
     "babel-plugin-transform-react-jsx": "^6.24.1",
     "bell-on-bundler-error-plugin": "^2.0.0",
-    "block-editor-hmr": "^0.6.1",
+    "block-editor-hmr": "^0.6.3",
     "chalk": "^4.1.0",
     "clean-webpack-plugin": "^3.0.0",
-    "copy-webpack-plugin": "^6.0.2",
-    "css-loader": "^4.3.0",
+    "copy-webpack-plugin": "^7.0.0",
+    "css-loader": "^5.0.1",
     "detect-port-alt": "^1.1.6",
     "eslint-loader": "^4.0.2",
-    "file-loader": "^6.0.0",
-    "inquirer": "^7.2.0",
+    "file-loader": "^6.2.0",
+    "inquirer": "^7.3.3",
     "is-root": "^2.1.0",
-    "mini-css-extract-plugin": "^0.11.2",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
-    "postcss-flexbugs-fixes": "^4.1.0",
-    "postcss-loader": "^3.0.0",
+    "mini-css-extract-plugin": "^1.3.3",
+    "optimize-css-assets-webpack-plugin": "^5.0.4",
+    "postcss": "^8.2.2",
+    "postcss-flexbugs-fixes": "^5.0.2",
+    "postcss-loader": "^4.1.0",
     "postcss-preset-env": "^6.7.0",
-    "sass-loader": "^10.0.2",
-    "signal-exit": "^3.0.2",
-    "style-loader": "^1.2.1",
-    "terser-webpack-plugin": "^4.2.0",
-    "ts-loader": "^8.0.1",
-    "url-loader": "^4.1.0",
-    "webpack-bundle-analyzer": "^3.8.0",
-    "webpack-fix-style-only-entries": "^0.5.0",
-    "webpack-manifest-plugin": "^2.0.4"
+    "sass-loader": "^10.1.0",
+    "signal-exit": "^3.0.3",
+    "style-loader": "^2.0.0",
+    "terser-webpack-plugin": "^5.0.3",
+    "ts-loader": "^8.0.13",
+    "url-loader": "^4.1.1",
+    "webpack-bundle-analyzer": "^4.3.0",
+    "webpack-fix-style-only-entries": "^0.6.0",
+    "webpack-manifest-plugin": "^3.0.0"
   },
   "peerDependencies": {
     "sass": "*",
     "typescript": "*",
```

Comment:
Can we move `typescript` (and `eslint`) to `peerDependenciesMeta` instead? Both these libraries are actually **optional** dependencies, and everything should work just fine without them..?

```json
  "peerDependenciesMeta": {
    "eslint": {
      "optional": true
    },
    "typescript": {
      "optional": true
    }
  }
```

Also, `eslint-loader` has been deprecated, and [`eslint-webpack-plugin`](https://github.com/webpack-contrib/eslint-webpack-plugin#migrate-from-eslint-loader) is the almost-swap-in replacement.

I only mention these as quite a few other package updates (only indirectly related to webpack) are included in this PR, too.